<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data</class>
 <widget class="QMainWindow" name="Data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Результаты экзаменов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>160</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="letter1">
       <property name="placeholderText">
        <string>Литра класса</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="subject">
       <property name="placeholderText">
        <string>Предмет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="month">
       <property name="placeholderText">
        <string>Месяц</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="print_table">
       <property name="text">
        <string>Вывести таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_table_b">
       <property name="text">
        <string>Сохранить таблицу</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="back_b2">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваша таблица:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>581</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип сортировки:</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="name">
       <property name="text">
        <string>Фамилия Имя</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="percent">
       <property name="text">
        <string> Процент выполнения</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QRadioButton" name="increase">
       <property name="text">
        <string>По возрастанию</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="decrease">
       <property name="text">
        <string>По убыванию</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
